src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -2280,6 +2280,9 @@ cldnn::network::ptr primitive_inst::get_unfused_subgraph() {
         t.add_primitive(std::const_pointer_cast<primitive>(_node->get_primitive()));
         outer_dep_ids.push_back(_node->id());
 
+        primitive_id fused_bias_eltw = _node->is_type<fully_connected>() ?
```

Comment:
I don't understand why we need this `fused_bias_eltw ` thing and all related changes. Could you explain why having extra check in is_valid_fusion() method is not enough?

Reply:
When bias is fused into fc, a new fc primitive with bias is created as below:
https://github.com/openvinotoolkit/openvino/blob/91744e16053403a23d33fd0a00b02584604b37ed/src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp#L399-L404
and, the fused eltwise primitive for bias is not included in fused_desc.
So, if (fc + add (for bias) + mul) are unfused, the input for mul cannot be found in the unfused graph.
To handle this case, I added the `fused_bias_eltw`.

![image](https://github.com/user-attachments/assets/cd9a2fc5-6628-419e-aa54-d998c80d2b88)

Reply:
Do you need to unfuse bias too?

Reply:
No, I don't need to unfuse bias. I just wanted to connect `mul` to `fc` to which `add` was fused.

Reply:
Then why do you need to save eltwise_add (bias) name? You don't need it to reconstruct the unfused graph

Reply:
As seen in the above image, there is no `add` layer in the unfused graph. There are only two layers, `fc` and `mul` in the unfused graph. The fused fc node does not have any information about `add`. So, when we add `mul` into the unfused graph, we cannot find the `add` that was the input of the `mul`. But, we know that the output of `fc` should be connected to `mul` instead of the missing `add`. At this point, we need the `fused_bias_eltw` in order to let it know that `fc` is the `add` that was the input of `mul`. In this example, `fused_bias_eltw` has string `add`.

Reply:
The purpose of `fused_bias_eltw` is to let `mul` know that `fc` is `add`.

Reply:
Can't you just check that post-op if the first one, and then just set input[0] as original fused_node? Or maybe something a little bit more complicated, i.e. if dep is not outer && fused-op id == 0, then set this inner-dep as fused_node

Reply:
Oh! let me try that. Thank you.

Reply:
In the case of SD3, it's not enough just to check if fused-op id ==0. Could you let me know how to check dep is outer or not?

Reply:
As you reviewed, I removed `fused_bias_eltw` and updated to check `fd.deps`. Thank you.